tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -447,6 +447,24 @@ public function provideTestCases()
                     'operators' => ['|' => BinaryOperatorSpacesFixer::NO_SPACE],
                 ],
             ],
+            [
+                '<?php
+$arr = array_map(function ($i): array {
+    $a  = 1;
+    $aa = 1;
+
+    return [];
+}, [1, 2, 3]);',
+
+                '<?php
+$arr = array_map(function ($i): array {
```

Comment:
Please add test case where instead of `function (` there is `function /* foo */ (`.